### PR TITLE
manager: add missing manager_listener_broker_uri

### PR DESCRIPTION
**Pull request by berendt**
This will fix the listener service

Signed-off-by: Christian Berendt <berendt@osism.tech>

### CODE REVIEW

File: environments/manager/secrets.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ---
 netbox_api_token: "0000000000000000000000000000000000000000"
+
+manager_listener_broker_password: BO6yGAAq9eqA7IKqeBdtAEO7aJuNu4zfbhtnRo8Y
```
Comment (osfrickler) [inline environments/manager/secrets.yml:4]: Can we use "password" here like in other locations to make it obvious that this is not secure and only for testing?

Comment (berendt) [reply environments/manager/secrets.yml:4]: This is the password from the Rabbitmq account in OpenStack. It's fine for me to changes that. But I would prefer to do this in a separate PR.

**Review by osfrickler — COMMENTED** [review]
Oh, if I had known that this is the same account that is used by OpenStack, I wouldn't have asked for this change, because now you can longer see that these two secrets are in fact the same.

I would question however that it is a good idea to have the same secret needing to be specified in multiple locations. If these cannot share the same secret store, this is IMO an indication that one wants to use dedicated accounts.

I'm also wondering why you only reference node-0 in the broker_url, shouldn't all three controllers be listed?

**Comment by berendt** [comment]
Separate RabbitMQ user for python-osism comes separately. I had tested the monitoring user from Kolla. It had too few rights.

**Comment by berendt** [comment]
Another issue now:

```
amqp.exceptions.AccessRefused: (0, 0): (403) ACCESS_REFUSED - Login was refused using authentication mechanism PLAIN. For details see the broker logfile.
```

**Comment by berendt** [comment]
The format Kombu needs is different from oslo.messaging. Works now.

```
2022-03-13 18:39:57 - Connected to amqp://openstack:**@192.168.16.10:5672//
2022-03-13 18:40:11 - Connection to broker lost, trying to re-establish connection...
[...]
2022-03-13 18:40:11 - Connected to amqp://openstack:**@192.168.16.11:5672//
```

**Review by osfrickler — APPROVED** [review]

**Pull request merged**